dao-contracts/src/voting/governance_voting.rs

```diff
@@ -49,6 +45,7 @@ pub trait GovernanceVotingTrait {
 pub struct GovernanceVoting {
     variable_repo: Variable<Option<Address>>,
     reputation_token: Variable<Option<Address>>,
+    va_token: Variable<Option<Address>>,
```

Comment:
The naming is inappropriate, in the context of kyc voting, the address points to the kyc token, not the va one. So the name should be more generic.